src/Compilers/VisualBasic/Portable/BoundTree/Expression.vb

```diff
@@ -109,7 +122,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         Private ReadOnly Property IHasOperatorMethodExpression_UsesOperatorMethod As Boolean Implements IHasOperatorMethodExpression.UsesOperatorMethod
             Get
                 If ExpressionKind() = OperationKind.CompoundAssignmentExpression Then
-                    Return TypeOf Me.Right Is BoundUserDefinedBinaryOperator
+                    Dim rightBinary As IBinaryOperatorExpression = TryCast(Me.Right, IBinaryOperatorExpression)
```

Comment:
That can lead to incorrect behavior in that something can appear to be invalid that is actually valid, but it doesn't crash, and so isn't necessarily an Update 2 issue. This needs to be correct, but it doesn't need to be correct now.